<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #161A22;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>570</y>
      <width>1671</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane {
 background: linear-gradient(135deg, #0F111A 0%, #1A1C2A 100%);
 border: 1px solid #2A2D3A;
 border-radius: 10px;
}
QTabBar::tab {
 background: #1E1F2D;
 color: #B0BEC5;
 padding: 10px 20px;
 margin: 2px;
 border-radius: 8px;
 font-size: 14px;
}
QTabBar::tab:selected {
 background: #0288D1;
 color: #FFFFFF;
}
QTabBar::tab:hover {
 background: #2A2D3A;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Stats</string>
     </attribute>
     <widget class="QWidget" name="statsWidget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1621</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: linear-gradient(135deg, #0F111A 0%, #1A1C2A 100%);
border-radius: 10px;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="currentUserPhotoLabel">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>30</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1E1F2D;
border: 1px solid #3A3D4A;
border-radius: 	30px;
box-shadow: 0 4px 10px rgba(0, 0, 0, 0.2);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>1851</width>
      <height>971</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22252D;
border-radius:40px;
border: 1px solid #FFFFFF; /* Subtle blue-gray border */
</string>
    </property>
    <widget class="QPushButton" name="employeeButton">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>90</y>
       <width>200</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border-radius: 5px; /* Slightly rounded corners for a modern look */</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/img/employee.png</normaloff>:/img/employee.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>500</width>
       <height>700</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="coachButton">
     <property name="geometry">
      <rect>
       <x>1470</x>
       <y>90</y>
       <width>200</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border-radius: 5px; /* Slightly rounded corners for a modern look */</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/img/COACH.png</normaloff>:/img/COACH.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>500</width>
       <height>700</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="dashboard_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;</string>
    </property>
    <property name="text">
     <string>Competition</string>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;</string>
    </property>
    <property name="text">
     <string>Match</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/Vector.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;</string>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/Group 1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>34</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/Group 3.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/logo_football.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>620</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;</string>
    </property>
    <property name="text">
     <string>Team</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>28</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/Group 2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>920</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>920</y>
      <width>28</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/logout.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>790</y>
      <width>135</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #393C44;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>1867</x>
      <y>60</y>
      <width>16</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 1px solid transparent ; /* Subtle border matching chart borders */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/img/Vectorrrr.png</normaloff>:/img/Vectorrrr.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>471</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Speedo</family>
      <pointsize>29</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>Manage your

users</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>561</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans Nova</family>
      <pointsize>21</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: #B1B1B1;</string>
    </property>
    <property name="text">
     <string>Add new users easily and assign roles : 
Coach or Employee.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>220</y>
      <width>140</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images.qrc">:/img/ADDBACKG.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="add_user">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>230</y>
      <width>141</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-radius: 5px; /* Slightly rounded corners for a modern look */</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/img/ADDICON.png</normaloff>:/img/ADDICON.png</iconset>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>200</y>
      <width>1</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(168, 168, 168);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>900</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(45deg, #0288D1 0%, #03A9F4 100%);
border-radius: 30px;
transition: background 0.3s, transform 0.2s;
}
QPushButton:hover {
 background: linear-gradient(45deg, #03A9F4 0%, #4FC3F7 100%);
 transform: scale(1.1);
}
QPushButton:pressed {
 background: linear-gradient(45deg, #01579B 0%, #0288D1 100%);
 transform: scale(0.9);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/logout.png</normaloff>:/images/logout.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="profileButton">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(45deg, #0288D1 0%, #03A9F4 100%);
border-radius: 30px;
transition: background 0.3s, transform 0.2s;
}
QPushButton:hover {
 background: linear-gradient(45deg, #03A9F4 0%, #4FC3F7 100%);
 transform: scale(1.1);
}
QPushButton:pressed {
 background: linear-gradient(45deg, #01579B 0%, #0288D1 100%);
 transform: scale(0.9);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/logout.png</normaloff>:/images/logout.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="teamButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(45deg, #0288D1 0%, #03A9F4 100%);
border-radius: 30px;
transition: background 0.3s, transform 0.2s;
}
QPushButton:hover {
 background: linear-gradient(45deg, #03A9F4 0%, #4FC3F7 100%);
 transform: scale(1.1);
}
QPushButton:pressed {
 background: linear-gradient(45deg, #01579B 0%, #0288D1 100%);
 transform: scale(0.9);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/logout.png</normaloff>:/images/logout.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_9">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:white;</string>
    </property>
    <property name="text">
     <string>User name</string>
    </property>
   </widget>
   <widget class="QLabel" name="dashboard_8">
    <property name="geometry">
     <rect>
      <x>1710</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Franklin Gothic Book</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:#393C44;</string>
    </property>
    <property name="text">
     <string>Role</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playerButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(45deg, #0288D1 0%, #03A9F4 100%);
border-radius: 30px;
transition: background 0.3s, transform 0.2s;
}
QPushButton:hover {
 background: linear-gradient(45deg, #03A9F4 0%, #4FC3F7 100%);
 transform: scale(1.1);
}
QPushButton:pressed {
 background: linear-gradient(45deg, #01579B 0%, #0288D1 100%);
 transform: scale(0.9);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/logout.png</normaloff>:/images/logout.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="compButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: linear-gradient(45deg, #0288D1 0%, #03A9F4 100%);
border-radius: 30px;
transition: background 0.3s, transform 0.2s;
}
QPushButton:hover {
 background: linear-gradient(45deg, #03A9F4 0%, #4FC3F7 100%);
 transform: scale(1.1);
}
QPushButton:pressed {
 background: linear-gradient(45deg, #01579B 0%, #0288D1 100%);
 transform: scale(0.9);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/images/logout.png</normaloff>:/images/logout.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <zorder>widget_3</zorder>
   <zorder>tabWidget</zorder>
   <zorder>currentUserPhotoLabel</zorder>
   <zorder>dashboard_7</zorder>
   <zorder>dashboard_3</zorder>
   <zorder>label_17</zorder>
   <zorder>dashboard_2</zorder>
   <zorder>label_18</zorder>
   <zorder>label_19</zorder>
   <zorder>label</zorder>
   <zorder>dashboard_4</zorder>
   <zorder>label_21</zorder>
   <zorder>dashboard_6</zorder>
   <zorder>label_22</zorder>
   <zorder>line</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>label_20</zorder>
   <zorder>label_23</zorder>
   <zorder>label_2</zorder>
   <zorder>add_user</zorder>
   <zorder>line_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>profileButton</zorder>
   <zorder>teamButton</zorder>
   <zorder>dashboard_9</zorder>
   <zorder>dashboard_8</zorder>
   <zorder>playerButton</zorder>
   <zorder>compButton</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
